packages/design-system/src/components/contextMenu/menu.js

```diff
@@ -290,14 +290,18 @@ const Menu = ({
   ]);
 
   return (
-    <MenuWrapper ref={wrapperRef} isIconMenu={isIconMenu} role="menu">
+    <MenuWrapper
+      ref={wrapperRef}
+      isIconMenu={isIconMenu}
+      role="menu"
+      {...props}
+    >
       <MenuList
         data-testid="context-menu-list"
         ref={listRef}
         isIconMenu={isIconMenu}
         aria-label={groupLabel}
         role="group"
-        {...props}
```

Comment:
Moved props up one layer here so that props like `className` and `onMouseDown` apply to the outer container